src/ethereum_test_tools/exceptions/exceptions.py

```diff
@@ -147,6 +147,10 @@ class BlockException(ExceptionBase):
     """
     Block's excess blob gas in header is incorrect.
     """
+    RLP_ENCODING = auto()
+    """
```

Comment:
We should also add `RLP_ENCODING` to `class TransactionException` and use that for tests that define the exception at the `Transaction` object.

We would then also the need to check like:
```python
if TransactionException.RLP_ENCODING in block.exception or BlockException.RLP_ENCODING in block.exception:
   ...
```

Reply:
I emphasized on that this exception is about ethereum structures not RLP being invalid